scripts/parse_tools/__init__.py

```diff
@@ -32,7 +32,7 @@
 from parse_source   import ParseSyntaxError, ParseInternalError
 from parse_source   import CCPPError, context_string
 from parse_object   import ParseObject
-from parse_checkers import check_fortran_id, FORTRAN_ID
+from parse_checkers import check_fortran_id, LITERAL, FORTRAN_ID
```

Comment:
I do not really like this name (`LITERAL`) because it only covers integers. Can it be changed to `LITERAL_INT` or something similar?

Reply:
Will use LITERAL_INT

Reply:
Thanks, sorry for being picky.